### ISSUE TITLE
Excerpt meta box: placeholders should not be used as replacement for labels

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1371]
Opening for discussion and because there's already a pending issue on core Trac, see https://core.trac.wordpress.org/ticket/40331

The excerpt textarea uses an `aria-label="Excerpt"` attribute, and that's a good thing although visible labels are always preferable.

It also uses a placeholder attribute:
`placeholder="Write an excerpt (optional)"`
this is against the spec and not ideal for accessibility. According to the spec, placeholder attributes should be used exclusively to give users a "hint" or "suggestion" about the expected data format.

Please refer to the core Trac ticket for more details.
